gcloud/bigquery/job.py

```diff
@@ -990,10 +1061,13 @@ def _build_resource(self):
             },
             'configuration': {
                 self._JOB_TYPE: {
-                    'query': self.query,
+                    'query': self.query
```

Comment:
Please leave trailing commas in place for Python code:  the JSON serializer will drop them for us, and they make maintaining the code much simpler.